include/netutils/webclient.h

```diff
@@ -165,6 +165,7 @@ typedef CODE int (*webclient_body_callback_t)(
     FAR void *ctx);
 
 struct webclient_tls_connection;
+struct webclient_poll_info;
```

Comment:
```suggestion
struct webclient_tls_connection_s;
struct webclient_poll_info_s;
```
According to the coding style, structure names must end with the suffix `_s`.

Reply:
ok. but i will not mix style fixes into this PR.